protocols/superrare/overview.md

```diff
@@ -0,0 +1 @@
+Overview of protocol and governance process.
```

Comment:
Hey @jasminesabio , It seems that the Overview file wasn't updated, Can you add it and try again ?